src/main/java/lotto/view/LottoPurchaseListView.java

```diff
@@ -0,0 +1,16 @@
+package lotto.view;
+
+import lotto.domain.Lottos;
+
+public class LottoPurchaseListView {
+
+    private LottoPurchaseListView() {
+    }
+
+    public static void print(Lottos lottos) {
+        System.out.print(lottos.purchaseNumber() + "개를 구매했습니다.");
+        System.out.println();
+
+        lottos.purchaseList().stream().forEach(System.out::println);
```

Comment:
foreach 사용 시에는 stream을 사용하지 않으셔도 괜찮습니다 :)

```suggestion
        lottos.purchaseList().forEach(System.out::println);
```